build.cake

```diff
@@ -22,156 +20,78 @@ var signingCertificatePassword = Argument("signing_certificate_password", "");
-});
+Task("Restore")
+	.IsDependentOn("Clean")
+    .Does(() => DotNetCoreRestore("source"));
 
-Task("__ZipNET45TestProject")
+Task("Build")
+    .IsDependentOn("Restore")
     .Does(() => {
-        Zip("source/Calamari.Tests/bin/Release/net451/win7-x64/", Path.Combine(artifactsDir, "Binaries.zip"));
-    });
-
-Task("__Test")
-    .Does(() =>
-{
-    var settings =  new DotNetCoreTestSettings
-    {
-        Configuration = configuration
-    };
-
-    if(!string.IsNullOrEmpty(framework))
-        settings.Framework = framework;  
-
-    if(!string.IsNullOrEmpty(testFilter))
-        settings.ArgumentCustomization = f => {
-            f.Append("-where");
-            f.AppendQuoted(testFilter);
-            return f;
-        };
-
-     DotNetCoreTest("source/Calamari.Tests/project.json", settings);
-});
-
-Task("__Pack")
-    .Does(() =>
-{
-    DoPackage("Calamari", "net40", nugetVersion);
-    DoPackage("Calamari.Azure", "net45", nugetVersion);   
-});
-
-private void DoPackage(string project, string framework, string version)
-{ 
-	var outputDirectory = Path.Combine(artifactsDir, project);
-	
-    DotNetCorePublish(Path.Combine("./source", project), new DotNetCorePublishSettings
-    {
-        Configuration = configuration,
-        OutputDirectory = outputDirectory,
-        Framework = framework,
-		NoBuild = true
-    });
-
-	SignBinaries(outputDirectory);
+		 DotNetCoreBuild("./source", new DotNetCoreBuildSettings
+		{
+			Configuration = configuration
+		});
+	});
+
+Task("Test")
+    .IsDependentOn("Build")
+    .Does(() => {
+		var projects = GetFiles("./source/**/*Tests.csproj");
+		foreach(var project in projects)
+			DotNetCoreTest(project.FullPath, new DotNetCoreTestSettings
+			{
+				Configuration = configuration,
+				NoBuild = true,
+				ArgumentCustomization = args => {
+					if(!string.IsNullOrEmpty(testFilter)) {
+						args = args.Append("--where").AppendQuoted(testFilter);
+					}
+					return args.Append("--logger:trx");
+				}
+			});
+	});
 	
-    TransformConfig(Path.Combine(artifactsDir, project, "project.json"), new TransformationCollection {
-        { "version", version }
-    });
-
-    DotNetCorePack(Path.Combine(artifactsDir, project), new DotNetCorePackSettings
-    {
-        OutputDirectory = artifactsDir,
-        NoBuild = true
-    });
-
-    DeleteDirectory(Path.Combine(artifactsDir, project), true);
-    DeleteFiles(artifactsDir + "*symbols*");
-}
-
-private void SignBinaries(string outputDirectory)
-{
-	var files = GetFiles(outputDirectory + "/*.exe");
-	var signtoolPath = MakeAbsolute(File("./source/Tools/signtool/signtool.exe"));
-
-	Sign(files, new SignToolSignSettings {
-			ToolPath = signtoolPath,
-            TimeStampUri = new Uri("http://timestamp.globalsign.com/scripts/timestamp.dll"),
-            CertPath = signingCertificatePath,
-            Password = signingCertificatePassword
-    });
-}
-
-Task("__Publish")
-    .WithCriteria(isContinuousIntegrationBuild)
+Task("Publish")

```

Comment:
This task can be removed altogether, it was never called in the previous script. TC is currently pushing packages for us after all the platform tests pass